src/batou/lib/supervisor.py

```diff
@@ -82,23 +83,31 @@ def verify(self):
             return
         for dependency in self.dependencies:
             dependency.assert_no_changes()
-        out, err = self.ctl('status {}'.format(self.name))
-        if 'RUNNING' not in out:
+        running = self.is_running()
+        if (self.enable and not running) or (not self.enable and running):
```

Comment:
Hah, one of the few places where a XOR would be applicable. ;) Please ignore me...